uint8 ask
---
uint32 width
uint32 height
float64 Fov
float64 nearRange
float64 maxRange
